core/src/main/java/com/datastax/oss/driver/internal/core/resolver/UnknownHostResponse.java

```diff
@@ -0,0 +1,17 @@
+package com.datastax.oss.driver.internal.core.resolver;
```

Comment:
Let's move it to `core/src/main/java/com/datastax/oss/driver/internal/core/resolver/MockResolver/`